README.md

```diff
@@ -127,5 +127,22 @@ with context.new_entry_span(op=str('https://github.com/apache/skywalking')) as s
 
 Check [the FAQ page](docs/FAQ.md) or add the FAQs there.
 
+## For Developers
+
+### Steps to get an operational virtual environment:
+
+1. `git clone https://github.com/apache/skywalking-python.git`
+2. `cd skywalking-python/scripts` (*make sure you actually go into the directory since the scripts use relative paths*)
```

Comment:
nit: I hope the script itself can tackle the paths by some extra work, that would be more friendly to the developers